docs/extension.md

```diff
@@ -86,3 +90,92 @@ class ExtensionExample {
     }
 }
 ```
+
+### Access your Extension's service in Spring Boot
+In Spring Boot, your extension's service is _lazily_ initialized. It means that you must annotate your injected extension's service
+field with the `@Lazy` annotation in addition to the `@Autowired` one:
+```java
+@Configuration
+public class MyBdkExtensionConfig {
+
+  @Bean
+  public MyBdkExtension myBdkExtension() {
+    return new MyBdkExtension();
+  }
+}
+
+@RestController
+@RequestMapping("/api")
+public class ApiController {
+
+    @Lazy // required, otherwise Spring Boot application startup will fail
```

Comment:
can't we put the lazy on the bean instead?

Reply:
We can't actually, since [the extension service bean is dynamically registered in the application content](https://github.com/finos/symphony-bdk-java/pull/620/files#diff-4fd58766e7a40ef56f93bcc4ce0405d5ead73f8edadb2b1bbfd15221a79b8052R62).